development/bot/waitress_server.py

```diff
@@ -0,0 +1,6 @@
+from waitress import serve
+import vk
+import telegram
+
+serve(vk.py, host='0.0.0.0', port=8080)
+serve(telegram.py, host='0.0.0.0', port=8080)
```

Comment:
Are you sure that you can run both instances on the same port? Have you tested it?

Reply:
Does not work. How i can fix it?